mirgecom/array_context.py

```diff
@@ -227,11 +235,16 @@ def _check_gpu_oversubscription(actx: ArrayContext) -> None:
 
 def log_disk_cache_config(actx: ArrayContext) -> None:
     """Log the disk cache configuration."""
-    from mpi4py import MPI
+    import sys
 
     assert isinstance(actx, (PyOpenCLArrayContext, PytatoPyOpenCLArrayContext))
 
-    rank = MPI.COMM_WORLD.Get_rank()
+    if "mpi4py" in sys.modules:
+        from mpi4py import MPI
+        rank = MPI.COMM_WORLD.Get_rank()
+    else:
+        rank = 0
+
```

Comment:
Can you check if it's an MPI array context and get `actx.mpi_communicator`?

Reply:
Done in [16a3a6a](https://github.com/illinois-ceesd/mirgecom/pull/1060/commits/16a3a6a9560db47c01b18bcac13ad366a10f8c04)